templates/diff_per_rule_bootstrap.php

```diff
@@ -35,13 +35,52 @@ function template_diff_per_rule_bootstrap(array $report) {
         crossorigin="anonymous">
     <script
         src="https://code.jquery.com/jquery-2.2.4.min.js"
+		integrity="sha384-rY/jv8mMhqDabXSo+UCggqKtdmBfd3qC2/KvyTDNQ6PcUJXaxK1tMepoQda4g5vB"
         crossorigin="anonymous">
     </script> 
     <script
         src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
         integrity="sha384-Tc5IQib027qvyjSMfHjOMaLkfuWVxZxUPnCJA7l2mCWNIpG9mGCD8wGNIcPD7Txa"
         crossorigin="anonymous">
-    </script> 
+    </script>
+	<script>
```

Comment:
Please reindent using spaces. I try to stick to more standard conventions in my private projects.